[IMP] iot: Add part regarding screen (#2730)
odoo/documentation: content/applications/productivity/iot/devices/screen.rst

   :align: center
   :alt: An example of a PoS (point of sale) order on a screen display.
   
.. note::
   Access the customer display from any other computer by going to the :abbr:`IoT (Internet of Things)`
   Box homepage and clicking on the :guilabel:`POS Display` button.

Connection
==========

The way to connect the screen display to the :abbr:`IoT (Internet of Things)` Box differs depending
on the model.

.. tabs::

   .. tab:: IoT Box model 4

      Connect up to two screens with Micro-HDMI cables on the side of the :abbr:`IoT (Internet of Things)`
      Box. If two screens are connected, they can display distinct content (see usage below).

   .. tab:: IoT Box model 3

      Connect the screen with an HDMI cable on the side of the :abbr:`IoT (Internet of Things)` Box.

.. important::
   Screen(s) should be connected before the :abbr:`IoT (Internet of Things)` Box is switched on. If it
   is already on, connect the screen(s), and then restart the :abbr:`IoT (Internet of Things)` Box by
   unplugging it and plugging it back into its power source.

.. warning::
   The usage of HDMI/Micro-HDMI adapters may cause issues which will result in a blank, black screen on

>    the screen display. Cable usage is recommended.

Reviewer: "Cable usage is recommended"? I'm not sure to understand that. "Correct cable usage" maybe?

================
Connect a Screen
================

When using your *IoT Box* in Odoo, you could need to use a screen.
After being connected, it will allow you to display to your client the
current PoS order.

.. image:: screen/screen-pos-client-display.png
   :align: center

.. note::
        You can also access the Customer Display from any other computer by
        accessing the *IoT Box* homepage and clicking on the *POS Display* button.

.. note::
        Starting version 15. The usage of the *IoT Box* is no longer mandatory
        to display the PoS session. You can set in the PoS configuration

>         to display the PoS session from any computer using the PoS.

Reviewer 2: I agree with ANV that we should remove the mention of Odoo 15. 

Since this PR is targeting the `13.0` branch, it would be odd to mention Odoo 15. After the forward-ports are complete, we could open a new PR that targets the `15.0` branch to add this `note` about Odoo 15.

Connection
==========

> The way to connect the screen to the *IoT Box* differ depending on the model.

Reviewer 2: ```suggestion
The way to connect the screen display to the :abbr:`IoT (Internet of Things)` Box differs depending
on the model.
```

.. tabs::

>    .. tab:: *IoT Box model 4*
> 
>         You can connect up to 2 screens with Micro-HDMI cables on the side of the *IoT Box*.
>         If 2 screens are connected, they can display distinct content (see usage below).

Reviewer 2: ```suggestion
   .. tab:: IoT Box model 4

      Connect up to two screens with Micro-HDMI cables on the side of the :abbr:`IoT (Internet of Things)`
      Box. If two screens are connected, they can display distinct content (see usage below).
```

>    .. tab:: *IoT Box model 3*
> 
>         Connect the screen with a HDMI cable on the side of the *IoT Box*

Reviewer 2: ```suggestion
   .. tab:: IoT Box model 3

      Connect the screen with an HDMI cable on the side of the :abbr:`IoT (Internet of Things)` Box.
```

.. important::

>     Screen(s) should be connected before the *IoT Box* is switched on.
>     If it is already on, connect the screen(s) then restart the *IoT Box*
>     by unplugging and plug back its power source.

Reviewer 2: ```suggestion
   Screen(s) should be connected before the :abbr:`IoT (Internet of Things)` Box is switched on. If it
   is already on, connect the screen(s), and then restart the :abbr:`IoT (Internet of Things) Box by
   unplugging it and plugging it back into its power source.
```

.. warning::

>     The usage of HDMI/Micro-HDMI adapters may cause issue which will result on
>     a black screen on the screen. Cable usage is recommended.

Reviewer 2: ```suggestion
   The usage of HDMI/Micro-HDMI adapters may cause issues which will result in a blank, black screen on
   the screen display. Cable usage is recommended.
```

> If the connection was successful, the screen should display the
> "POS Client display" screen.

Reviewer 2: ```suggestion
If the connection was successful, the screen should display the :guilabel:`POS Client display`
screen.
```

.. image:: screen/screen-pos-client-display-no-order.png
   :align: center

> 

Reviewer 2: ```suggestion
   :alt: The default "POS Client Display" screen that appears when a screen display is successfully
   connected to an IoT Box.
   
```

> You should also see your screen in the list of Displays
> on the *IoT Box* homepage.

Reviewer 2: ```suggestion
The screen should also appear in the list of :guilabel:`Displays` on the :abbr:`IoT (Internet of
Things)` Box homepage.
```

.. image:: screen/screen-screen-name-example.png
   :align: center

> 

Reviewer 2: ```suggestion
   :alt: An example of a screen display name shown on the IoT Box homepage.
   
```

.. note::

>     If no screen is detected, a default display will be used named "Distant Display"

Reviewer 2: ```suggestion
    If no screen is detected, a default display named :guilabel:`Distant Display` will be used instead.
```

    .. image:: screen/screen-no-screen.png
       :align: center

> 

Reviewer 2: ```suggestion
   :alt: The "Distant Display" screen name will be used if no screen is detected.
   
```

Usage
=====

> Show Point of Sales orders to customers

Reviewer 2: ```suggestion

Show Point of Sales orders to customers
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> To use the screen in your *Point of Sale* app, go to :menuselection:`Point of Sale --> Configuration
> --> Point of Sale`, open the one you want to configure, then click on *Edit* and enable the *IoT
> Box Devices* feature.

Reviewer 2: ```suggestion
To use the screen in the :guilabel:`Point of Sale` app, go to :menuselection:`Point of Sale -->
Configuration --> Point of Sale`, open the desired screen, click on :guilabel:`Edit`, and enable the
:guilabel:`IoT Box Devices` feature.
```

> Now, choose the *IoT Box* in the dropdown menu and choose the screen in *Customer Display* option.
> Then, you hit save.

Reviewer 2: ```suggestion
Now, select :guilabel:`IoT Box` in the drop-down menu and choose the screen in the
:guilabel:`Customer Display` option. Then, click :guilabel:`Save`.
```

    .. image:: screen/screen-pos-screen-config.png
       :align: center

> 

Reviewer 2: ```suggestion
   :alt: Connect the screen display to the Point of Sale app.
   
```

> The screen is now be available in your PoS sessions. A "screen" icon will appear in the top bar to inform the
> connection status with the screen.

Reviewer 2: ```suggestion
The screen is now available for :abbr:`PoS (Point of Sale)` sessions. A "screen" icon will appear
in the top bar to inform the connection status with the screen.
```

    .. image:: screen/screen-pos-icon.png
       :align: center

> 

Reviewer 2: ```suggestion
   :alt: The "screen" icon on the Point of Sale display shows the connection status with the screen.
```

> The screen will automatically show the PoS orders and update when changes are performed on the order.

Reviewer 2: ```suggestion
The screen will automatically show the :abbr:`PoS (Point of Sale) orders and update when changes are
performed on the order.
```

.. image:: screen/screen-pos-client-display.png
   :align: center

> 

Reviewer 2: ```suggestion
   :alt: An example of a PoS order on a screen display.
```

Display a website
~~~~~~~~~~~~~~~~~

> Opening the screen form view at :menuselection:`IoT --> Devices --> (screen device)`
> will allow you to choose a particular website URL to display on the screen with the
> *Screen URL* field.

Reviewer 2: ```suggestion
Opening the screen form view at :menuselection:`IoT --> Devices --> (screen device)` allows the user
to choose a particular website URL to display on the screen with the :guilabel:`Screen URL` field.
```

.. image:: screen/screen-pos-client-display-no-order.png
   :align: center
   :alt: The default "POS Client Display" screen that appears when a screen display is successfully

>    connected to an IoT Box.

Author: ```
iot/devices/screen.rst:46: WARNING: Error in "image" directive:
invalid option block.

.. image:: screen/screen-pos-client-display-no-order.png
   :align: center
   :alt: The default "POS Client Display" screen that appears when a screen display is successfully
   connected to an IoT Box.
```
-> the text must be aligned with the option content and not the option

    .. image:: screen/screen-pos-screen-config.png
       :align: center
   :alt: Connect the screen display to the Point of Sale app.
       :alt: Connect the screen display to the Point of Sale app.
   
The screen is now available for :abbr:`PoS (Point of Sale)` sessions. A "screen" icon will appear
in the top bar to inform the connection status with the screen.

    .. image:: screen/screen-pos-icon.png
       :align: center
   :alt: The "screen" icon on the Point of Sale display shows the connection status with the screen.

> The screen will automatically show the :abbr:`PoS (Point of Sale) orders and update when changes are

Author: need to have a new line when a block content is over
